### ISSUE TITLE
"make test" failing! Test serializing more strings than the capacity of the initial strings table.

### ISSUE THREAD

**Issue by jonathanpmartins** [igbinary/igbinary#82]
Build complete.
Don't forget to run 'make test'.

```
=====================================================================
PHP         : /usr/bin/php7.0 
PHP_SAPI    : cli
PHP_VERSION : 7.0.12-1+deb.sury.org~xenial+1
ZEND_VERSION: 3.0.0
PHP_OS      : Linux - Linux ubuntu-pc 4.4.0-42-generic #62-Ubuntu SMP Fri Oct 7 23:11:45 UTC 2016 x86_64
INI actual  : /work/trash/igbinary/tmp-php.ini
More .INIs  :   
CWD         : /work/trash/igbinary
Extra dirs  : 
VALGRIND    : Not used
=====================================================================
TIME START 2016-10-17 21:54:54
=====================================================================
PASS Check for igbinary presence [tests/igbinary_001.phpt] 
PASS Check for null serialisation [tests/igbinary_002.phpt] 
PASS Check for bool serialisation [tests/igbinary_003.phpt] 
PASS Check for integer serialisation [tests/igbinary_004.phpt] 
PASS Check for double serialisation [tests/igbinary_005.phpt] 
PASS Check for simple string serialization [tests/igbinary_006.phpt] 
PASS Check for simple array serialization [tests/igbinary_007.phpt] 
PASS Check for array+string serialization [tests/igbinary_008.phpt] 
PASS Check for reference serialisation [tests/igbinary_009.phpt] 
PASS Check for reference serialization (Original example, not using var_dump) [tests/igbinary_009b.phpt] 
PASS Array test [tests/igbinary_010.phpt] 
PASS Object test [tests/igbinary_012.phpt] 
PASS Object-Array test [tests/igbinary_013.phpt] 
PASS Object-Reference test [tests/igbinary_014.phpt] 
PASS Check for serialization handler [tests/igbinary_015.phpt] 
PASS Check for serialization handler, ini-directive [tests/igbinary_015b.phpt] 
PASS Object test, __sleep [tests/igbinary_016.phpt] 
PASS Object test, __wakeup [tests/igbinary_017.phpt] 
PASS Object test, __sleep error cases [tests/igbinary_018.phpt] 
PASS Object test, __autoload [tests/igbinary_019.phpt] 
PASS Object test, incomplete class [tests/igbinary_020.phpt] 
PASS Object Serializable interface [tests/igbinary_021.phpt] 
PASS Object test, unserialize_callback_func [tests/igbinary_022.phpt] 
PASS Resource [tests/igbinary_023.phpt] 
PASS Recursive objects [tests/igbinary_024.phpt] 
PASS Object test, array of objects with __sleep [tests/igbinary_025.phpt] 
PASS Cyclic array test [tests/igbinary_026.phpt] 
PASS Cyclic array test 2 [tests/igbinary_026b.phpt] 
PASS Check for serialization handler [tests/igbinary_027.phpt] 
PASS Serialize object into session, full set [tests/igbinary_028.phpt] 
PASS Igbinary module info [tests/igbinary_029.phpt] 
PASS Unserialize invalid data [tests/igbinary_030.phpt] 
PASS Object Serializable interface throws exceptions [tests/igbinary_031.phpt] 
PASS Object test, __sleep and __wakeup exceptions [tests/igbinary_032.phpt] 
PASS Object test, cyclic references [tests/igbinary_033.phpt] 
PASS Unserialize invalid random data [tests/igbinary_034.phpt] 
PASS b0rked random data test [tests/igbinary_040.phpt] 
PASS Check for double NaN, Inf, -Inf, 0, and -0. IEEE 754 doubles [tests/igbinary_041.phpt] 
PASS Closure [tests/igbinary_042.phpt] 
PASS Object serialization with compact strings [tests/igbinary_043.phpt] 
PASS Check for double extremes [tests/igbinary_044.phpt] 
SKIP APC serializer registration [tests/igbinary_045.phpt] reason: APC not loaded
SKIP APCu serializer registration [tests/igbinary_045b.phpt] reason: APCu not loaded
SKIP APCu serializer registration - more data types [tests/igbinary_045c.phpt] reason: APCu not loaded
PASS Correctly unserialize scalar refs. [tests/igbinary_046.phpt] 
PASS Correctly unserialize multiple object refs. [tests/igbinary_046b.phpt] 
PASS Correctly unserialize multiple array refs. [tests/igbinary_046c.phpt] 
PASS Correctly unserialize multiple object refs and non-refs. [tests/igbinary_046d.phpt] 
PASS Check for serialization handler, SessionHandlerInterface [tests/igbinary_047.phpt] 
PASS Object test, __set not called for private attr in extended class [tests/igbinary_048.phpt] 
PASS Object test, __set not called for private attr in extended class [tests/igbinary_048b.phpt] 
PASS Correctly unserialize multiple references in arrays [tests/igbinary_049.phpt] 
PASS Correctly unserialize multiple references in objects [tests/igbinary_049b.phpt] 
PASS Correctly unserialize cyclic object references [tests/igbinary_050.phpt] 
PASS Object test, __wakeup (With multiple references) [tests/igbinary_051.phpt] 
PASS Object Serializable interface can be serialized in references [tests/igbinary_052.phpt] 
PASS __wakeup can modify properties without affecting other objects [tests/igbinary_053.phpt] 
PASS __wakeup can add dynamic properties without affecting other objects [tests/igbinary_054.phpt] 
PASS __wakeup can replace a copy of the object referring to the root node. [tests/igbinary_055.phpt] 
FAIL Test serializing more strings than the capacity of the initial strings table. [tests/igbinary_057.phpt] 
PASS Should not call __destruct if __wakeup throws an exception [tests/igbinary_058.phpt] 
PASS Should not call __destruct if __wakeup throws an exception (in arrays) [tests/igbinary_058b.phpt] 
PASS igbinary_unserialize should never convert from packed array to hash when references exist (Bug #48) [tests/igbinary_059.phpt] 
PASS Nested objects cause segfault, php bug #54662 [tests/igbinary_bug54662.phpt] 
PASS igbinary_unserialize causes segfault on 3rd call for objects with dynamic property [tests/igbinary_bug72134.phpt] 
PASS Unserialize backwards compatible with v1. [tests/igbinary_unserialize_v1_compatible.phpt] 
=====================================================================
TIME END 2016-10-17 21:54:55

=====================================================================
TEST RESULT SUMMARY
---------------------------------------------------------------------
Exts skipped    :    0
Exts tested     :   14
---------------------------------------------------------------------

Number of tests :   66                63
Tests skipped   :    3 (  4.5%) --------
Tests warned    :    0 (  0.0%) (  0.0%)
Tests failed    :    1 (  1.5%) (  1.6%)
Expected fail   :    0 (  0.0%) (  0.0%)
Tests passed    :   62 ( 93.9%) ( 98.4%)
---------------------------------------------------------------------
Time taken      :    1 seconds
=====================================================================

=====================================================================
FAILED TEST SUMMARY
---------------------------------------------------------------------
Test serializing more strings than the capacity of the initial strings table. [tests/igbinary_057.phpt]
=====================================================================
```

**Comment by jonathanpmartins** [comment]
Sorry for the styles, I just copy and paste the test from the terminal.

**Comment by TysonAndre** [comment]
This isn't anything to worry about in the code itself, the JSON extension just wasn't available for the test in recent patch versions of php 7.0

Fixed in 6a2d5b7 by using implode instead of json.

**Comment by jonathanpmartins** [comment]
Thanks @TysonAndre
